languages/shop-standards-de_DE.po

```diff
@@ -171,110 +179,114 @@ msgstr ""
 msgid "Shop Standards"
 msgstr ""
 
-#: src/WooCommerce.php:29
+#: src/WooCommerce.php:35 src/WooCommerce.php:683
+msgid "Marketing focus product"
+msgstr "Marketing focus product"
+
+#: src/WooCommerce.php:36 src/WooCommerce.php:659
+msgid "Price comparison focus product"
+msgstr "Repricing: kein automatisches Repricing"
+
+#: src/WooCommerce.php:37 src/WooCommerce.php:671
+msgid "Price comparison lower only product"
+msgstr "Repricing: Preise nur senken"
```

Comment:
Please keep the English label in code in line, so that developers understand what this checkbox is doing.
```suggestion
msgid "Repricing: Only lower prices"
msgstr "Repricing: Preise nur senken"
```